### ISSUE TITLE
Netlify 404 page

### ISSUE THREAD

**Issue by ann-kilzer** [ourjapanlife/findadoc-frontend#129]
Use our own 404 page instead of the Netlify one

For example, https://www.findadoc.jp/bees goes to the Netlify page

**Comment by RossellaFer** [comment]
Docs on Nuxt.js file system routing: [create a _.vue file](https://nuxtjs.org/docs/2.x/features/file-system-routing#unknown-dynamic-nested-routes) in the /pages directory

**Comment by claudiofreitas** [comment]
@RossellaFer Thank you, I tried to follow the link you sent and it was useful! 
I made an attempt and seems to be working.
Maybe I can have someone double check just in case.

My attempt code can be visualized here:
https://github.com/ourjapanlife/findadoc-frontend/compare/main...claudiofreitas:129-netlify-404-page?expand=1

Document I tried to follow:
https://nuxtjs.org/docs/2.x/deployment/netlify-deployment#how-to-deploy-on-netlify-

The page is obviously super crude, just a test message to check the poc.
The deployed result can be checked here:
https://peaceful-chandrasekhar-3eb8f9.netlify.app/mitsukaranai
